src/Rules/UnitExtendsFromTestCaseRule.php

```diff
@@ -43,7 +43,7 @@ public function processNode(Node $node, Scope $scope): array
         $node = $node;
 
         $extendedClass = $node->extends;
-        if ($extendedClass === null || ! UnitTestRuleHelper::isUnitTest((string) $scope->getNamespace(), (string) $node->name, $this->unitTestNamespaceContainsString)) {
+        if ($extendedClass === null || ! UnitTestRuleHelper::isUnitTest((string) $scope->getNamespace(), (string) $node->name, $this->unitTestNamespace)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'ikvasnica\PHPStan\Rules\UnitTestRuleHelper' in method 'processNode'.](https://app.codacy.com/manual/ikvasnica/phpstan-clean-test/pullRequest?prid=4930471)